src/workerd/api/node/tests/sys-nodejs-test.wd-test

```diff
@@ -0,0 +1,15 @@
+using Workerd = import "/workerd/workerd.capnp";
+
+const unitTests :Workerd.Config = (
+  services = [
+    ( name = "nodejs-sys-test",
+      worker = (
+        modules = [
+          (name = "worker", esModule = embed "sys-nodejs-test.js")
+        ],
+        compatibilityDate = "2023-10-01",
+        compatibilityFlags = ["nodejs_compat_v2"],
```

Comment:
question: should be use `nodejs_compat_v2` or just `nodejs_compat`?

Reply:
I would set the compat date forward to at least `2024-09-23` and just use `nodejs_compat`